app/html/sub/post.html

```diff
@@ -323,7 +330,38 @@ <h4>@{_('Comment preview')}</h4>
   @else:
     <div class="comments">
       <h2 id="cmnts" data-cnt="@{post['comments']}"><a href="@{url_for('sub.view_post', sub=sub['name'], pid=post['pid'])}">@{_('%(comments)i comments', comments=post['comments'])}</a></h2>
-      @{pcomm.renderComments(post, subInfo, subMods, comments, highlight)!!html}
+      @if len(comments) > 1:
+      <div class="pure-menu pure-menu-horizontal">
+        <ul class="pure-menu-list">
+          <li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover">
+            <span id="sortby" class="pure-menu-link">@{_('Sort by')}</span>
```

Comment:
```suggestion
            <span id="sortby" class="pure-menu-link">@{_('Sort by: <b>%(currsort)s</b>', currsort=_('top') if sort == 'top' else _('new'))!!html}</span>
```

It's probably better to show the current sort right away